tools/wpt/testfiles.py

```diff
@@ -190,7 +193,8 @@ def exclude_ignored(files: Iterable[Text], ignore_rules: Optional[Sequence[Text]
     return changed, ignored
 
 
-def files_changed(revish: Text,
+def files_changed(repo_root: Text,
+                  revish: Text,
                   ignore_rules: Optional[Sequence[Text]] = None,
                   include_uncommitted: bool = False,
                   include_new: bool = False
```

Comment:
How about:

```suggestion
def files_changed(revish: Text,
                  ignore_rules: Optional[Sequence[Text]] = None,
                  include_uncommitted: bool = False,
                  include_new: bool = False,
                  repo_root: Text = wpt_root,
```

Then there's no need to update the `run_changed_files()`, `run_tests_affected()`, `wpt/run.py`, and `ci/jobs.py` callsites.

Also, then there's no need for the `repo_root or wpt_root` expression below, since the caller can just omit `files_changed(repo_root=...)`.